weave/trace_server/trace_server_interface.py

```diff
@@ -251,6 +251,20 @@ class CallsDeleteRes(BaseModel):
     pass
 
 
+class CallsDescendantsReq(BaseModel):
+    project_id: str
+    call_ids: list[str]
+    limit: Optional[int] = None
+    # list of columns to select, defaults to all columns
```

Comment:
Instead of writing this as a comment, make it a field description.  This additionally gives us nice documentation when it's exported in the OpenAPI spec